src/index.js

```diff
@@ -514,7 +532,7 @@ class ServerlessAppsyncPlugin {
           Type: 'AWS::AppSync::ApiCache',
           Properties: {
             ApiCachingBehavior: config.caching.behavior,
-            ApiId: config.apiId || { 'Fn::GetAtt': [logicalIdGraphQLApi, 'ApiId'] },
+            ApiId: { 'Fn::GetAtt': [logicalIdGraphQLApi, 'ApiId'] },
```

Comment:
@anastyn Why did you remove support for Api Keys and Caching?
As far as I can tell, someone might want to re-use an existing API but still deploy new API keys or configure caching.
This is definitely something possible and that was possible before this change.
I think it is useful to keep support and also not making any breaking change.